### PR TITLE
chore(docs): add new script from latest ios release

### PR DESCRIPTION
**Pull request by ghost**
From https://newrelic.slack.com/archives/C0DSGL3FZ/p1695225878781039

### CODE REVIEW

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx [hunk 1]
```diff
@@ -77,9 +77,17 @@ These procedures to configure your iOS/tvOS app with CocoaPods and Objective-C a
     if [ ${CONFIGURATION} = "Debug" ]; then
         echo "Skipping DSYM upload CONFIGURATION: ${CONFIGURATION}"
         exit 0
-    fi
+    fi
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx:80]: @cdillard-NewRelic Is this a typo or is it fine?

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx [hunk 2]
```diff
@@ -97,7 +105,7 @@ These procedures to configure your iOS/tvOS app with CocoaPods and Swift also ap
    ```
 3. Open your project in Xcode by running this command from the terminal in your project directory:
 
-   ```shell
+   ```swi
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx:108]:
```suggestion
   ```shell
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx [hunk 3]
```diff
@@ -118,23 +126,31 @@ These procedures to configure your iOS/tvOS app with CocoaPods and Swift also ap
 6. Add a build script to your target's build phases. Ensure the new build script is the very last build script. Then paste the following, replacing `APP_TOKEN` with your [application token](/docs/mobile-apps/viewing-your-application-token):
      
  - For iOS agent 7.4.0 or higher:
-   ```shell
+   ```swift
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx:129]:
```suggestion
   ```shell
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx [hunk 4]
```diff
@@ -118,23 +126,31 @@ These procedures to configure your iOS/tvOS app with CocoaPods and Swift also ap
...
    SCRIPT=`/usr/bin/find "${SRCROOT}" -name run-symbol-tool | head -n 1`
    /bin/sh "${SCRIPT}" "APP_TOKEN"
    ```
  - For iOS agent 7.3.8 or lower:
-   ```shell
+   ```swift
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx:134]:
```suggestion
   ```shell
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx [hunk 5]
```diff
@@ -118,23 +126,31 @@ These procedures to configure your iOS/tvOS app with CocoaPods and Swift also ap
...
    SCRIPT=`/usr/bin/find "${SRCROOT}" -name newrelic_postbuild.sh | head -n 1`
    /bin/sh "${SCRIPT}" "APP_TOKEN"
    ```
  - OPTIONAL: Add the following lines to your build script above the existing lines to skip symbol upload during debugging:
-    ```shell
+    ```swift
```
Comment (ally-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx:139]:
```suggestion
    ```shell
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx (hunk 1 above)
Comment (cdillard-NewRelic) [reply src/content/docs/mobile-monitoring/new-relic-mobile-ios/installation/cocoapods-installation.mdx:80]: its fine, in shell scripts if statements look this way.

**Review by nbaenam — DISMISSED** [review]
LGTM!

**Pull request merged**